tests/read.time.estimator.test.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import readTimeEstimator from '../src/helpers/read.time.estimator';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Read time estimator', () => {
+  it('SHould return the estimated time', (done) => {
+    const time1 = readTimeEstimator('This text is supposed to be the body of an article <img');
+    const time2 = readTimeEstimator('This text is supposed to be the body of an article <img <img');
+    const time3 = readTimeEstimator('This text is supposed to be the body of an article <img <img <img');
```

Comment:
@nkpremices instead of just checking that `readTimeEstimator` returns a number, a better test would be to check that two texts or two articles with the same length have the same read time.

Something like:
text1 = 'An apple is a fruit'
text2 = 'What is your name?'

expect(readTimeEstimator(text1)).to.eql(readTimeEstimator(text2))

Reply:
let me do that @MemunaHaruna